interface/main/calendar/find_group_popup.php

```diff
@@ -0,0 +1,306 @@
+            padding: 3px;
+        }
+
+        #searchResultsHeader {
+            width: 100%;
+            background-color: lightgrey;
+        }
+
+        #searchResultsHeader table {
+            width: 96%; /* not 100% because the 'searchResults' table has a scrollbar */
+            border-collapse: collapse;
+        }
+
+        #searchResultsHeader th {
+            font-size: 0.7em;
+        }
+
+        #searchResults {
+            width: 96%;
+            height: 80%;
+            overflow: auto;
+        }
+
+        #results_table{
+            text-align: center;
+        }
+
+        /* search results column widths */
+        .srName {
+            width: 30%;
+        }
+
+        .srGID {
+            width: 21%;
+        }
+
+        .srType {
+            width: 17%;
+        }
+
+        .srStartDate {
+            width: 17%;
+        }
+
+        #searchResults table {
+            width: 100%;
+            border-collapse: collapse;
+            background-color: white;
+        }
+
+        #searchResults tr {
+            cursor: hand;
+            cursor: pointer;
+        }
+
+        #searchResults td {
+            font-size: 0.7em;
+            border-bottom: 1px solid #eee;
+        }
+
+        .oneResult {
+        }
+
+        .billing {
+            color: red;
+            font-weight: bold;
+        }
+
+        /* for search results or 'searching' notification */
+        #searchstatus {
+            font-size: 0.8em;
+            font-weight: bold;
+            padding: 1px 1px 10px 1px;
+            font-style: italic;
+            color: black;
+            text-align: center;
+        }
+
+        .noResults {
+            background-color: #ccc;
+        }
+
+        .tooManyResults {
+            background-color: #fc0;
+        }
+
+        .howManyResults {
+            background-color: #9f6;
+        }
+
+        #searchspinner {
+            display: inline;
+            visibility: hidden;
+        }
+
+        /* highlight for the mouse-over */
+        .highlight {
+            background-color: #336699;
+            color: white;
+        }
+    </style>
+
+    <script type="text/javascript"
+            src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-2-2/index.js"></script>
+    <!-- ViSolve: Verify the noresult parameter -->
+    <?php
+    if (isset($_GET["res"])) {
+        echo '
+<script language="Javascript">
+			// Pass the variable to parent hidden type and submit
+			opener.document.theform.resname.value = "noresult";
+			opener.document.theform.submit();
+			// Close the window
+			window.self.close();
+</script>';
+    }
+    ?>
+    <!-- ViSolve: Verify the noresult parameter -->
+
+    <script language="JavaScript">
+
+        function selgid(gid, name) {
+            if (opener.closed || !opener.setgroup)
+                alert("<?php echo htmlspecialchars(xl('The destination form was closed; I cannot act on your selection.'), ENT_QUOTES); ?>");
```

Comment:
change to xls(since is a javascript literal):
``````
alert("<?php echo xls('The destination form was closed; I cannot act on your selection.'); ?>");